lib/Donors/Donors.js

```diff
@@ -12,10 +13,19 @@ import { defaultColumnMapping } from './constants';
 import { DonorsContainer } from './DonorsContainer';
 import { useFetchDonors } from './hooks';
 
-export function Donors({ name, donorOrganizationIds, ...rest }) {
+export function Donors({ name, donorOrganizationIds, onChange, ...rest }) {
   const [donorIds, setDonorIds] = useState(donorOrganizationIds);
   const { donors, isLoading } = useFetchDonors(donorIds);
```

Comment:
Could you please apply some changes to improve the UX (keep prev changes in the MCL and display loading indicator inside the list):
```
// useFetchDonors.js

export const useFetchDonors = (donorOrganizationIds = DEFAULT_DATA, options = {}) => {
 ..
    {
       enabled: Boolean(donorOrganizationIds.length)
       ...options,
     },
  ..
  );

// ------
// Donors.js

..

  const { donors, isFetching } = useFetchDonors(donorIds, { keepPreviousData: true });

..

  // Remove this
  if (isLoading) {
    return <Loading />;
  }

..

  <FieldArray
     ..
     component={DonorsContainer}
     loading={isFetching}
     ..
   />

```

Reply:
This is a very nice improvement. Thank you for your advice.